src/System.Net.Sockets/tests/FunctionalTests/ArgumentValidationTests.cs

```diff
@@ -503,6 +503,7 @@ public void Select_LargeList_Throws_ArgumentOutOfRange()
             Assert.Throws<ArgumentOutOfRangeException>(() => Socket.Select(null, null, largeList, -1));
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Bug in AcceptAsync that dereferences null SAEA argument")]
```

Comment:
> But does that mean that the test is being disabled due to a bug in NETFX and not a test issue?

Yes, e.g. in this particular one you commented on, netfx is missing argument validation on the argument and just dereferences it immediately, so with netfx you get a NullReferenceException, whereas with core you get an ArgumentNullException.